README.md

```diff
@@ -1,24 +1,183 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: `rails db:create, rails db:migrate, rails db:seed`. 👍